openlibrary/api.py

```diff
@@ -192,13 +191,12 @@ def unlimited_query(q):
         if 'limit' in q and q['limit'] == False:
             return unlimited_query(q)
         else:
-            q = json.dumps(q)
-            response = self._request("/query.json?" + urllib.parse.urlencode(dict(query=q)))
-            return unmarshal(json.loads(response.read()))
+            response = self._request("/query.json", params=dict(query=json.dumps(q)))
+            return unmarshal(response.json())
 
     def import_ocaid(self, ocaid, require_marc=True):
         data = {'identifier': ocaid, 'require_marc': 'true' if require_marc else 'false'}
-        return self._request('/api/import/ia', method='POST', data=urllib.parse.urlencode(data)).read()
+        return self._request('/api/import/ia', method='POST', data=data).text
```

Comment:
Like above, this is a change from returning bytes to returning str.

Reply:
Changed from ```text``` to ```content``` here too.